### PR TITLE
Feat: develop explore page

### PR DESCRIPTION
**Pull request by deeseeker**
This pull request implements the Explore Page for the project, developed with careful consideration for responsiveness across all screen sizes. The implementation follows a modular approach to ensure maintainability and scalability by organizing reusable parts into well-defined components.

Features and Enhancements:

1. **Responsive Design Across All Screen Sizes:**

- Ensured the layout adapts seamlessly from mobile to desktop.

- Used media queries and responsive design best practices to enhance user experience.

- Verified the design against common breakpoints (e.g., 320px, 768px, 1024px, 1440px).

2. **File Structure and Componentization:**

-  **Folder Structure:**

   Organized files under specific folders to separate concerns clearly.

- /components: Contains reusable UI components.

- Reusable Components:
Extracted commonly used elements (e.g., Card, CategoryList) to ensure a DRY (Don't Repeat Yourself) approach. resolves issue #65

### CODE REVIEW

**Comment by stephanniegb** [comment]
Hey, thanks for you contributions! It looks great

**Comment by stephanniegb** [comment]
clicking the cards does nothing, it is supposed to link to the campaign detail page

File: public/kosuki.jpg
Comment (stephanniegb) [inline public/kosuki.jpg:1]: What are these images for? the cards in the figma are dummy cards and should not be added to the page

File: app/utils/explore-data.ts
Comment (stephanniegb) [inline app/utils/explore-data.ts:1]: there is no need for the dummy data, just use the campaigns  being returned from the endpoint

**Review by stephanniegb — CHANGES_REQUESTED** [review]

**Comment by stephanniegb** [comment]
Screenshot 2024-12-21 at 8 52 14 PM
Screenshot 2024-12-21 at 8 52 20 PM
You are supposed to change the UI of the campaign cards, not create dummy campaigns.

File: public/kosuki.jpg
Comment (deeseeker) [reply public/kosuki.jpg:1]: I just came up with it through my initiative. I will factor in all the feedbacks. Sorry for the inconvenience

**Comment by stephanniegb** [comment]
LFTM🚀. Thanks for your contribution✨

**Pull request merged**